### ISSUE TITLE
Windows 1909

### ISSUE THREAD

**Issue by felixcruz3** [MicrosoftDocs/windows-itpro-docs#5536]
I have been using this process since it was published. The only thing I have changed was 1809 to 1909. The process no longer seems to function with 1909, it never seems to load the AutoPilot profile and just goes directly to Sign in with Microsoft page not my company sign in page as previous versions. Is anything else needed?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 17bfe927-f7b7-608b-8537-5c2274ccdced
* Version Independent ID: 5dd9abca-92d9-fbb2-6796-d1f6b9ab3c8b
* Content: [Windows Autopilot for existing devices](https://docs.microsoft.com/en-us/windows/deployment/windows-autopilot/existing-devices#feedback)
* Content Source: [windows/deployment/windows-autopilot/existing-devices.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/deployment/windows-autopilot/existing-devices.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @greg-lindsay
* Microsoft Alias: **greglin**

**Comment by Ry-77** [comment]
Having exactly the same problem.

**Comment by Gor3t3x** [comment]
Guys, have you seen this post by Michael? => https://oofhours.com/2019/09/19/a-challenge-with-windows-autopilot-for-existing-devices-and-windows-10-1903/
Seems to be for 1903, but it may still apply to 1909.

Btw, there are bugs in 1903 if you don't service (update) wim image (i use OSDBuilder) => https://oofhours.com/2019/09/15/windows-autopilot-known-issues-in-windows-10-1903/

With the "speeding up the process" on 1909 guide, i arrive on autopilot OOBE, but after entering credentials in the Autopilot OOBE page, i get a Windows Logon page between "Device Setup" and "Account Setup"  of the Enrollement Status page.

Maybe another bug when using a non-english version of Windows?

**Comment by illfated** [comment]
See also this update to the AutoPilot troubleshooting table:
- https://github.com/MicrosoftDocs/windows-itpro-docs/commit/35f088c0abfc0154fc38e42325d9c11f72834473

Rich Text Diff view of the commit:
- https://github.com/MicrosoftDocs/windows-itpro-docs/commit/35f088c0abfc0154fc38e42325d9c11f72834473?short_path=7a15321#diff-7a1532199b3669bedaba908b5b052be1

**Comment by felixcruz3** [comment]
@Gor3t3x
That article resolved my issue.

**Comment by Gor3t3x** [comment]
@felixcruz3
Are you using Enrollement Status Page during OOBE? If yes, don't you have a Windows logon prompt during it?

In 1903, i get "Welcome to XXX Corporation", i enter my credentials(usr+pwd), get enrollement status page, asks me for my biometrics… and i'm on the desktop.

In 1909, i get a strange Windows logon page, during the enrollement status page between "Device Configuration" and "account setup".

Are you experiencing the same? I'm using 1909 (french) from VLSC portal.

**Comment by felixcruz3** [comment]
@Gor3t3x
Yes, to both. 
Using 1903 and 1909 (en-us) from VLSC. We moved over 50 in the past 2 days mainly all 1909 and have not seen any additional screens nor encountered any odd behavior during oobe or esp after following the article. Also, we disabled Credential Guard back in June 2019, it was causing an oobe loop with AutoPilot.

I have multiple profiles for ESP. 
Default - All required device based apps; usually all of our 3rd party security apps. All other Profiles are Dynamic Groups [Departmental based] which target the user.
